### ISSUE TITLE
https://readthedocs.org accidentally tries to include content from http://localhost:5555 causing mixed content errors

### ISSUE THREAD

**Issue by jab** [rtfd/readthedocs.org#1324]
```
Mixed Content: The page at 'https://readthedocs.org/projects/bidict/versions/#'
was loaded over HTTPS, but requested an insecure stylesheet
'http://localhost:5555/static/css/public.css'. This request has been blocked;
the content must be served over HTTPS.

2015-06-06 17:22:18.916 (index):1 Mixed Content:
The page at 'https://readthedocs.org/projects/bidict/versions/#'
was loaded over HTTPS, but requested an insecure script
'http://localhost:5555/static/javascript/bundle-public.js'.
This request has been blocked; the content must be served over HTTPS.
```

Looks like references to development assets are accidentally sneaking past your build tool into production.

**Comment by jab** [comment]
![screen shot 2015-06-06 at 17 29 18](https://cloud.githubusercontent.com/assets/64992/8021532/960f163a-0c71-11e5-891e-4871f1d1bbcc.png)

**Comment by agjohnson** [comment]
Thanks for the report! Looks like @ericholscher closed this with fb793593. This should be resolved with our next deploy, if it isn't already.

**Comment by jab** [comment]
My pleasure! Looks like the fix hasn't been deployed yet. Any sense for when it will be?
